docs/xgcpolicy.md

```diff
@@ -25,21 +25,25 @@
 # -Xgcpolicy
 
 
-Controls the behavior of the garbage collector.
+Controls the behavior of the garbage collector by specifying different garbage collection policies.
 
 ## Syntax
 
         -Xgcpolicy:<parameter>
 
 ## Parameters
 
-Specify the garbage collection policy that you want the OpenJ9 VM to use:
-
-### `gencon`
+| Parameter                                                        | Default  |
+|------------------------------------------------------------------|----------|
+| [`balanced`](#balanced)                                          |          |
+| [`gencon`](#gencon)                                              | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">Default</span> |
+| [`metronome`](#metronome) (AIX<sup>&reg;</sup>, Linux x86 only)  |          |
+| [`optavgpause`](#optavdpause)                                    |          |
+| [`optthruput`](#optthruput)                                      |          |
+| [`subpool`](#subpool) (AIX, Linux, z/OS<sup>&reg;</sup> only)    |          |
```

Comment:
GC policy subpool is deprecated and it is an alias name for optthruput since Java 7.0. I believe we should remove any mention about it from OpenJ9 documentation

Reply:
If you go to the `subpool` section it does tell you that it is deprecated and an alias for `optthruput`. I thought that we had to leave deprecated options until they were removed and that was ok as long as they were marked so.  I'll update the top table for now to say `deprecated` against the option. Saves people clicking on the link and wasting time to find it marked deprecated later on.